client/app/components/Input/InputTag.jsx

```diff
@@ -103,13 +102,13 @@ export class InputTag extends React.Component<Props, State> {
     return checkboxWithLabel.length && checkboxWithLabel[0].id;
   };
 
-  onAutocompleteChange = (e: SyntheticEvent<HTMLInputElement>, { newValue }) => {
+  onAutocompleteChange = (e: SyntheticEvent<HTMLInputElement>, { newValue }: {newValue: string}) => {
     this.setState({
       autocompleteLabel: newValue,
     });
   };
 
-  onSelect = (event: SyntheticEvent<HTMLInputElement>, { suggestion, suggestionValue } ) => {
+  onSelect = (event: SyntheticEvent<HTMLInputElement>, { suggestion, suggestionValue }: {suggestion: string, suggestionValue: string} ) => {
```

Comment:
Over here the suggestionValue isn't needed so I will go ahead and remove that. But even still the issue will persist I believe. Don't really know what to do here.

Reply:
It looks like this error is happening because suggestion is a string therefore it can't have the label key. Should you be doing this on a `checkbox` instead?